blazegraph/Dockerfile

```diff
@@ -16,6 +21,21 @@ RUN --mount=type=cache,id=blazegraph-downloads-${TARGETARCH},sharing=locked,targ
     && \
     cleanup.sh
 
+# Remove old files & then install latest log4j-* files
+RUN --mount=type=cache,id=log4j-downloads-${TARGETARCH},sharing=locked,target=/opt/downloads \
+    download.sh \
+        --url "${LOG4J_URL}" \
+        --sha256 "${LOG4J_FILE_SHA256}" \
+    && \
+    ## Remove the outmoded log4j-* files that come with blazegraph
+    rm -f "/opt/tomcat/webapps/bigdata/WEB-INF/lib/log4j-1.2.17.jar" && \
```

Comment:
This line specifically removes `/opt/tomcat/webapps/bigdata/WEB-INF/lib/log4j-1.2.17.jar`

Reply:
@misilot The way this PR works is simply that we scrub that out after it is initially brought in by the Blazegraph install. So it's not surprising to me that the file is in an earlier layer but it is not in the final image.

Reply:
I think Line 11/16 is adding it. The two RUN commands I think need to be combined, so there isn't a layer where the log4j*.jar exists.

Reply:
Good idea - I'll try that...

Reply:
Ok, building now. I brought the RM back up to the first RUN...